adapters/aduptech/aduptech_test.go

```diff
@@ -0,0 +1,20 @@
+package aduptech
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/v3/adapters/adapterstest"
+	"github.com/prebid/prebid-server/v3/config"
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderAdUpTech, config.Adapter{
+		Endpoint: "https://example.com/rtb/bid"}, config.Server{ExternalUrl: "http://hosturl.com", GvlID: 1, DataCenter: "2"})
+
+	if buildErr != nil {
+		t.Fatalf("Builder returned unexpected error %v", buildErr)
+	}
```

Comment:
```suggestion
	if buildErr != nil {
		assert.NoError(t, buildErr, "Builder returned unexpected error")
	}
```

Reply:
Ah, good to know. This is our first time working in Go.

Reply:
For what it's worth, I realize you were calling fatal before so I realize you need the require.NoError - assertions from the assert module continue but the test fails whereas those from require halt immediately. This is kind of like google's gtest with its two variants for each assertion so when you can continue you report more/all things that fail but when you can't, you don't.